### PR TITLE
IPEX Duplicate importer V2

### PR DESCRIPTION
**Pull request by qiyuangong**
## Description

* Replace importer after importing ipex, then revert it back before converting model.
* Simply check to improve performance.

### 4. How to test?
- [ ] N/A
- [ ] Unit test
- [ ] Application test
- [ ] Document test
- [ ] ...

### 5. New dependencies

- [ ] New Python dependencies
       - Dependency1 
       - Dependency2
       - ...
- [ ] New Java/Scala dependencies and their license
       - Dependency1 and license1
       - Dependency2 and license2
       - ...

### CODE REVIEW

**Comment by qiyuangong** [comment]
Add calling package to avoid error when using intel_extension_for_deepspeed, e.g.,
```python
from intel_extension_for_deepspeed import XPU_Accelerator
```

Or
```python
import deepspeed
```

**Comment by qiyuangong** [comment]
@jason-dai added a control env `IPEX_DUPLICATE_CHECKER`, default is true. We can turn it off with `IPEX_DUPLICATE_CHECKER=1`

File: python/llm/src/ipex_llm/utils/ipex_importer.py [hunk 1]
```diff
@@ -18,10 +18,13 @@
 import logging
 import builtins
 import sys
+import os
 import inspect
 from ipex_llm.utils.common import log4Error
 
 
+# Default is True, set to False to disable IPEX duplicate checker
+IPEX_DUPLICATE_CHECKER = os.getenv("IPEX_DUPLICATE_CHECKER", 'True').lower() in ('true', '1', 't')
```
Comment (jason-dai) [inline python/llm/src/ipex_llm/utils/ipex_importer.py:27]: change to BIGDL_CHECK_DUPLICATE_IMPORT

Comment (qiyuangong) [reply python/llm/src/ipex_llm/utils/ipex_importer.py]: Done

**Comment by qiyuangong** [comment]
Failed unit tests are not related to this PR

**Pull request merged**